README.md

```diff
@@ -205,6 +205,7 @@ Aside from those listed here, many other apps and libraries can be easily be fou
 - [vector](https://github.com/vectordotdev/vector) - A high-performance observability data pipeline.
 - [vincenzo](https://github.com/gabrieldemian/vincenzo) - A bittorrent client for the terminal with vim-like keybindings.
 - [wireman](https://github.com/preiter93/wireman) - A gRPC client for the terminal.
+- [YADB](https://github.com/izya4ka/yadb) - A web directory brute-forcing tool.
```

Comment:
Let's keep the lexical order :)

Reply:
No I mean the list should be in alphabetical order. So you should move the entry up a bit